pyproject.toml

```diff
@@ -63,7 +66,6 @@ canmatrix = "0.9.1"
 click = "7.1.2"
 
 [tool.poetry.dev-dependencies]
-poetry-dynamic-versioning = "^0.13.1"
```

Comment:
Need to apply this line at the bottom of this file:
`build-backend = "poetry_dynamic_versioning.backend"`